tests/e2e/specs/backend/active-filters.test.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import {
+	getAllBlocks,
+	switchUserToAdmin,
+	getEditedPostContent,
+} from '@wordpress/e2e-test-utils';
+import { clearAndFillInput } from '@woocommerce/e2e-utils';
+import { insertBlockDontWaitForInsertClose } from '../../utils.js';
+import { visitBlockPage } from '@woocommerce/blocks-test-utils';
+
+const block = {
+	name: 'Active Product Filters',
+	slug: 'woocommerce/active-filters',
+	class: '.wc-block-active-filters',
+};
+
+describe( `${ block.name } Block`, () => {
+	beforeAll( async () => {
+		await switchUserToAdmin();
+		await visitBlockPage( `${ block.name } Block` );
+	} );
+
+	it( 'can be inserted once', async () => {
+		await insertBlockDontWaitForInsertClose( block.name );
+		expect( await getAllBlocks() ).toHaveLength( 1 );
+	} );
+
+	it( 'renders without crashing', async () => {
+		await expect( page ).toRenderBlock( block );
+	} );
+
+	it( 'can change title label in editor', async () => {
+		await expect( page ).toFill(
+			'.wc-block-active-filters__title > textarea',
+			'Current filters'
+		);
+
+		await clearAndFillInput(
+			'.wc-block-active-filters__title > textarea',
+			'Active filters'
+		);
+
+		expect( await getEditedPostContent() ).toMatchSnapshot();
```

Comment:
We don't usually use snapshot testing in our E2E tests. If you look in the snapshot file the new title does not appear there. 
I recommend you remove this line and search for the title again and write an assertion that the block title now has the title you provided `expect( ... ).toHaveTextContent( 'Current filters' );`.

Reply:
Thanks for pointing this out. I'll keep this in mind for future e2e tests. For now, I have closed this PR due to @opr drafted PR in https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4562.